qualitube/main.py

```diff
@@ -20,15 +20,8 @@ def chunks(lst: list, n) -> list:
         yield lst[i:i + n]
 
 def main():
-    logging.basicConfig(
-        level=logging.INFO,
-        handlers=[
-            logging.FileHandler('pipeline.log'),
-            logging.StreamHandler(sys.stdout)
-        ]
-    )
-
-    logging.info('Beginning of pipeline.')
+    """Main function."""
+    logger.info('Beginning of pipeline.')
```

Comment:
Here handlers must be added since you want logging messages to appear only in the CLI app:
```suggestion
    """Main function."""
    logger.info('Beginning of pipeline.')
    
    logger.addHandler(logging.FileHandler('pipeline.log'))
    logger.addHandler(logging.StreamHandler(sys.stdout))
```

Reply:
I didn't see you had already done a suggestion with the hotfix, sorry! I ended up isolating the logger setup in a function.